.changeset/orange-files-end.md

```diff
@@ -0,0 +1,8 @@
+---
+'@clerk/clerk-js': minor
+'@clerk/clerk-react': minor
+'@clerk/types': minor
+---
+
+Remove MemberRole Type`MemberRole` would always include the old role keys `admin`, `member`, `guest_member`. 
+If developers still depend on them after the introduction of custom roles, the can provide them as their custom types for authorization.
```

Comment:
🙃 Could you provide an example of how they would define the custom types for the old role keys?